FeedStoreChallenge/CoreDataManager/ManagedCache+CoreDataProperties.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ManagedCache+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Errol on 27/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+extension ManagedCache {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedCache> {
+		return NSFetchRequest<ManagedCache>(entityName: "ManagedCache")
+	}
+
+	@NSManaged public var timestamp: Date
+	@NSManaged public var feed: NSOrderedSet
```

Comment:
There's no need to make properties public since `ManagedCache` should be internal
Please remove `public` modifiers

```suggestion
	@NSManaged var timestamp: Date
	@NSManaged var feed: NSOrderedSet
```

Reply:
Done. Made it internal